framework/src/dm/Kconfig

```diff
@@ -7,6 +7,7 @@ config DM
 	bool "DM"
 	default n
 	depends on NET
+	depends on LIBC_NETDB
```

Comment:
Why do you use "depends on" instead of select?

Reply:
@sunghan-chang Because I can't sure LIBC_NETDB  has no dependency with other configs.
If LIBC_NETDB get a new dependcy I use 'select', same issue will occur again.

Reply:
You had better check whether there is a possibility making a dependancy or not. If it isn't, you had better use select.

Reply:
@sunghan-chang I changed 'depnds on' to 'select'